eegrasp/eegrasp.py

```diff
@@ -115,18 +133,20 @@ def compute_graph(self, W=None, epsilon=.5, sigma=.1):
         """
         Parameters
         ----------
-        W -> if W is passed, then the graph is computed. 
-        Otherwise the graph will be computed with self.W.
-        W should correspond to a non-sparse 2-D array.
-        Epsilon -> maximum distance to threshold the array.
-        sigma -> Sigma parameter for the gaussian kernel.
-
-        method: NN -> Nearest Neighbor
-                Gaussian -> Gaussian Kernel used based on the self.W matrix
+        W : numpy ndarray. 
+            if W is passed, then the graph is computed. Otherwise the graph will be computed with
+            `self.W`. `W` should correspond to a non-sparse 2-D array.
+        epsilon : float.
+            Maximum distance to threshold the array.
+        sigma : float.
+            Sigma parameter for the gaussian kernel.
+        method: string. 
+            Options are: "NN" or "Gaussian". Nearest Neighbor or Gaussian Kernel used based on the
+            `self.W` matrix respectively depending on the method used.
 
         Returns
         -------
-        G: Graph structure from PyGSP2        
+        G: Graph structure from PyGSP2.
```

Comment:
Better: "PyGSP2 graph structure"

Is not the same as a NetworkX structure?

Reply:
Nop, they are different. Though it is possible to convert a Pygsp2 graph object to a NetworkX structure